vm/tests/solutions/spec/ghost/files_spec.rb

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'spec_helper'
+
+describe file('/etc/sudoers.d/etc-sudoers.d-ghostuser.erb') do
```

Comment:
This is the name of the template file shouldn't we be checking
`/etc/sudoers.d/ghost_app` instead?

Reply:
Changed template name